tests/fast/EncryptionOps.toml

```diff
@@ -4,6 +4,9 @@ testClass = "Encryption"
 
 [[knobs]]
 enable_encryption = false
+enable_tlog_encryption = false
+enable_storage_server_encryption = false
+enable_blob_granule_encryption = false
```

Comment:
these are probably not needed after the knob changes?

Reply:
Good point will remove